core/trino-main/src/main/java/io/trino/cost/TaskCountEstimator.java

```diff
@@ -36,11 +36,11 @@
     @Inject
     public TaskCountEstimator(NodeSchedulerConfig nodeSchedulerConfig, InternalNodeManager nodeManager)
     {
-        requireNonNull(nodeSchedulerConfig, "nodeSchedulerConfig is null");
+        boolean schedulerIncludeCoordinator = requireNonNull(nodeSchedulerConfig, "nodeSchedulerConfig is null").isIncludeCoordinator();
```

Comment:
This is hard to read. Move the requireNonNull to a separate line (as it was before).

Reply:
no, it's not hard